docs/03-policies/prodev.md

```diff
@@ -0,0 +1,101 @@
+# Professional Development at CivicActions
+
+## Introduction
+
+CivicActions recognizes the importance of individual Professional Development for all team members. We value Professional Development for a number of reasons:
+
+* It’s a win-win. Professional Development grows the depth and value of the company offerings and enhances your career opportunities too.
+* Professional Development provides career mobility for each person - increasing our feeling of safety, which leads to:
+    * Having the courage to fail, which leads to experimentation, innovation, and learning
+    * Having the courage to challenge the status quo, which leads to organizational learning
+    * Having the courage to be vulnerable, which leads to authentic communications and connection
+    * Open-minded thinking, which leads to increased tolerance of others ideas and personality styles
+
+CivicActions supports and encourages the Professional Development (prodev) of each team member by providing an annual budget of $1,200 per person to be used for prodev-related expenses.
+
+In addition to the $1,200 annual budget, each team member is encouraged to use a “take what you need” approach to using time for prodev pursuits during business hours. If you need guidance on what amount of time is appropriate to spend on prodev in a given year, consult the table below.
+
+Historical Prodev Hours Spent: 
+
+| Year  | Company-wide Prodev Hours Used | # People Billing Prodev Hours | Average Prodev Hours Used Per Person Annually |
```

Comment:
@jmheaster for historical purposes, it might be useful to show how many people total were in the company at the time and percentage

Reply:
@nerdstein I like the idea of using participation percentage as a KPI for prodev and will add it to the appendix section for later use (assuming we can find historical data on total people in the company in a given year).  I would not include that participation data in the above table, as it is meant to be a guide for people on how much prodev time is typically used by people.

Reply:
Cool